src/entity/mongoRepository.ts

```diff
@@ -22,8 +22,10 @@ export async function createEntityRepository(
   return {
     getAllEntities: partial(getAllEntities, collection),
     findById: partial(findById, collection),
+    findByDid: partial(findByDid, collection),
     addEntity: partial(addEntity, collection),
     updateEntity: partial(updateEntity, collection),
+    deleteAll: partial(deleteAll, collection),
```

Comment:
Should we have `deleteAll` in production?

Reply:
It seems we are only using it for tests?

Reply:
Fair point. I removed it from repository interfaces and now droppping the whole db before each test. The tests still run quite fast.